src/applications/discharge-wizard/tests/v2/display-logic.unit.spec.js

```diff
@@ -8,13 +8,21 @@ describe('utilities: display logic', () => {
   describe('navigateForward', () => {
     describe('routing to discharge year', () => {
       const formResponses = { SERVICE_BRANCH: RESPONSES.ARMY };
+      const routeMap = ['introduction', 'service-branch'];
```

Comment:
There are a lot of strings for routes in this file; should we be using constants in case those get updated?